x-pack/test_serverless/api_integration/test_suites/security/config.ts

```diff
@@ -18,4 +18,23 @@ export default createTestConfig({
   // include settings from project controller
   // https://github.com/elastic/project-controller/blob/main/internal/project/security/config/elasticsearch.yml
   esServerArgs: ['xpack.ml.nlp.enabled=false'],
+
+  // Required for Fleet setup.
```

Comment:
Maybe it would make sense to extract this to a Fleet-specific config file?

Reply:
yes, perhaps Fleet related tests could move to `test_suites/fleet`?

Reply:
After some thought, I added new config files for Fleet in the `test_suites/observability/fleet` and `test_suites/security/fleet` folders in order to preserve the existing structure by project type and avoid adding unnecessary config to other tests. I also added a paragraph in the README on how to run these. Let me know how that sounds.

Reply:
Oh okay, I didn't realise it means the project type. I'm good with these changes 👍